src/ui/pages/leftSideMenu.page.ts

```diff
@@ -0,0 +1,20 @@
+import { SalesPortalPage } from "../pages/salesPortal.page.js";
+//import { ICredentials } from "../../data/types/creds.types.js";
+
+
+
+export class LeftSideMenu extends SalesPortalPage{
+    readonly uniqueElement = 'span.fs-4'
+
+    private readonly dropdownTitle = 'a#dropdownUser1';
+    private readonly productsLeftsideMenu = '[onclick="sideMenuClickHandler(\'Products\');"]';
+
+    async clickOnProductsLeftsideMenu () {
+        await this.click(this.productsLeftsideMenu)
+    }
```

Comment:
Можно сделать единый метод для всех кнопок в боковом меню:


```suggestion
    async clickOnMenuItem (itemName: MenuItems) {
        //Реализовать например через динамический xpath селектор, куда будет прокидываться имя кнопки в меню.
    }
```